src/bindings/c/src/common.h

```diff
@@ -224,3 +227,4 @@ struct mem_istream : virtual mem_stringbuf, std::istream {
 
 char* str_to_char_array(const std::string& str);
 ov::element::Type get_element_type(ov_element_type_e type);
+void dup_last_err_msg(const std::string msg);
```

Comment:
```suggestion
static inline void dup_last_err_msg(const std::string msg);
```

Reply:
This function need access 2 shared variables cross multiple source code files, inline cannot work.